tests/serverspec/default_spec.rb

```diff
@@ -48,10 +56,29 @@
   it { should be_enabled }
 end
 
+describe file(config_dir) do
+  it { should exist }
+  it { should be_directory }
+  it { should be_owned_by default_user }
+  it { should be_grouped_into default_group }
+  it { should be_mode 755 }
+end
+
+describe file("/no/such/dir") do
```

Comment:
the test is not necessary because the role has been modified not to create directories.